src/styles/settings/_override.build.scss

```diff
@@ -1,5 +1,2 @@
 // Set $ds-include-choice-error-highlight to automatically add an error highlight to choice components inside a ChoiceList with an errorMessage.
 $ds-include-choice-error-highlight: true;
-
-// Disabling the design system focus styles
-$ds-include-focus-styles: false;
```

Comment:
Remove the option to disable focus styles because we finally have a focus style to use!